Sources/OTelConventions/OTelAttribute.swift

```diff
@@ -0,0 +1 @@
+public enum OTelAttribute {}
```

Comment:
It'd be great to have a bit of documentation on this type given it's the main entry point to the API when using the String constants directly. Perhaps this documentation could also mention the existence of the more convenient extensions on `SpanAttribute`.

Reply:
Absolutely. I've added docs in https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/57d172d39bb68051967c3a5300517ad66322c179

Reply:
Thank you, looks good 👍

Reply:
Just to flag: This would need to be updated in case we decide to go with package traits.

Reply:
Good point. I updated the wording given the trait change here: https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/c21ef86c7cbaab502648141aa83ccdcd545ddf11#diff-a76c80f53c5e4fb345c1218700a0c6b3b2daeb962d16d3d0d36534f9ee0124d4L4-L6